hawc/apps/materialized/sql.py

```diff
@@ -27,7 +27,8 @@ class SQL(NamedTuple):
     ON scr.riskofbias_id = rob.id
     LEFT JOIN riskofbias_riskofbiasscoreoverrideobject ovr
     ON scr.id = ovr.score_id
-    WHERE rob.final AND rob.active;
+    WHERE rob.final AND rob.active
+    ORDER BY scr.id, ovr.id;
```

Comment:
this is the root issue - the order of creation of the materialized view wasn't deterministic; but this order by makes it so that it's always in the same order